src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -0,0 +1,35 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.dto.CartDto;
+import com.kodilla.ecommercee.dto.OrderDto;
+import org.springframework.web.bind.annotation.*;
```

Comment:
bleeee nie chcemy importów z gwiazdkami

Reply:
IntelliJ sam je łączy jeżeli jest więcej niż 3 importy z danego pakietu

Reply:
Można to zmienić i tak proponuję zrobić, ustawić w ustawieniach (Preferences > Editor > Code style > Java / Imports) te wartości na 999 (zarowno class count i names count/static)